clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -144,7 +144,7 @@ def ObjSizeOp : CIR_Op<"objsize", [Pure]> {
 
   let arguments = (ins CIR_PointerType:$ptr, SizeInfoType:$kind,
                    UnitAttr:$dynamic);
-  let results = (outs CIR_IntType:$result);
+  let results = (outs StdInt:$result);
```

Comment:
`Std` is a name that could have so many meanings in C and C++, let's try to avoid it. I suggest we use `PrimitiveInt`, `PrimitiveUInt`, `PrimitiveSInt`.

Reply:
Updated.